src/components/notebooks/create-notebook-card.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import Modal from '@/components/notebooks/modal';
+
+export default function CreateNotebookCard({
+  buttonTitle,
+  onSubmit,
+}: {
+  buttonTitle: string;
+  onSubmit: (title: string, description: string) => void;
+}) {
+  const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
I don't really like that the state of the modal is handled but this component. The modal should handle its own state. Then you could have some function to close or open the modal.

In cases where you need to open or close the modal, it should probably go through a function that abstract the logic (like `modal.open()`, `modal.close()`).

You really need to think about the modal as a class in a OO language. They need to be made with the same idea in mind.  

Related to that, `onClose` should have a default behaviour (closing the modal ofc).